sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/PruneFileSourcePartitionsSuite.scala

```diff
@@ -65,7 +65,8 @@ class PruneFileSourcePartitionsSuite extends QueryTest with SQLTestUtils with Te
           options = Map.empty)(sparkSession = spark)
 
         val logicalRelation = LogicalRelation(relation, tableMeta)
-        val query = Project(Seq('i, 'p), Filter('p === 1, logicalRelation)).analyze
+        val query = Project(Seq(Symbol("i"), Symbol("p")),
+          Filter(Symbol("p") === 1, logicalRelation)).analyze
```

Comment:
nit.
```scala
-        val query = Project(Seq(Symbol("i"), Symbol("p")),
-          Filter(Symbol("p") === 1, logicalRelation)).analyze
+        val query = Project(Seq($"i", $"p"), Filter($"p" === 1, logicalRelation)).analyze
```

Reply:
Here, I honestly wasn't sure whether I should swap out a Symbol for a Column reference. Where the test code is directly invoking catalyst primitives, I generally left it with a more conservative change, to just swap in Symbol. I imagine that a Column reference here could be fine too, but wanted to limit the scope of the change.

Reply:
Got it. No problem~